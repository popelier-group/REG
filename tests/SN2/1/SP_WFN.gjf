# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.39811000    0.00000500   -0.00017800
 H                 -0.70948900   -0.87904800   -0.54720000
 H                 -0.70948700    0.91322800   -0.48802300
 H                 -0.70967400   -0.03415400    1.03456100
 Br                 1.63552800   -0.00001200    0.00005600
 Cl                -3.49312500    0.00002500   -0.00004500

frame.wfn
