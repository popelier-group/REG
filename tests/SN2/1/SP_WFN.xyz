
6

C       -0.286587    0.004532    0.036834
H       -0.492312   -0.539184    0.948412
H       -0.719782   -0.476277   -0.829242
H       -0.574242    1.044129    0.110426
Br        1.730195   -0.028318   -0.222390
Cl       -3.355939    0.055015    0.431356
