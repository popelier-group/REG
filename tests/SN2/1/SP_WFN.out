 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.1/Gau-66356.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.1/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     66357.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.39811   0.00001  -0.00018 
 H                    -0.70949  -0.87905  -0.5472 
 H                    -0.70949   0.91323  -0.48802 
 H                    -0.70967  -0.03415   1.03456 
 Br                    1.63553  -0.00001   0.00006 
 Cl                   -3.49313   0.00003  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.398110    0.000005   -0.000178
      2          1           0       -0.709489   -0.879048   -0.547200
      3          1           0       -0.709487    0.913228   -0.488023
      4          1           0       -0.709674   -0.034154    1.034561
      5         35           0        1.635528   -0.000012    0.000056
      6         17           0       -3.493125    0.000025   -0.000045
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081168   0.000000
     3  H    1.081168   1.793253   0.000000
     4  H    1.081168   1.793269   1.793264   0.000000
     5  Br   2.033638   2.563454   2.563459   2.563462   0.000000
     6  Cl   3.095015   2.969979   2.969967   2.969710   5.128653
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.286587    0.004532    0.036834
      2          1           0       -0.492312   -0.539184    0.948412
      3          1           0       -0.719782   -0.476277   -0.829242
      4          1           0       -0.574242    1.044129    0.110426
      5         35           0        1.730195   -0.028318   -0.222390
      6         17           0       -3.355939    0.055015    0.431356
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.9353542           0.7861728           0.7861727
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       173.9344965657 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.63D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39487359     A.U. after   12 cycles
            NFock= 12  Conv=0.12D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.69188-101.25634 -62.34487 -56.16476 -56.16153
 Alpha  occ. eigenvalues --  -56.16153 -10.08373  -9.16929  -8.55423  -6.92794
 Alpha  occ. eigenvalues --   -6.92741  -6.92741  -6.38410  -6.37343  -6.37343
 Alpha  occ. eigenvalues --   -2.49523  -2.49221  -2.49221  -2.48375  -2.48375
 Alpha  occ. eigenvalues --   -0.66467  -0.54109  -0.50861  -0.30361  -0.30361
 Alpha  occ. eigenvalues --   -0.24579  -0.15934  -0.15934  -0.05729  -0.05568
 Alpha  occ. eigenvalues --   -0.05568
 Alpha virt. eigenvalues --    0.09532   0.10755   0.13920   0.13921   0.15939
 Alpha virt. eigenvalues --    0.17363   0.17953   0.17954   0.21515   0.21766
 Alpha virt. eigenvalues --    0.21767   0.24296   0.24297   0.24472   0.30759
 Alpha virt. eigenvalues --    0.30760   0.31399   0.33900   0.40397   0.50083
 Alpha virt. eigenvalues --    0.52026   0.52027   0.53779   0.53779   0.65529
 Alpha virt. eigenvalues --    0.67823   0.67823   0.71143   0.74461   0.74461
 Alpha virt. eigenvalues --    0.75549   0.76928   0.76928   0.82445   0.82445
 Alpha virt. eigenvalues --    0.84016   0.95315   1.07503   1.14998   1.14998
 Alpha virt. eigenvalues --    1.15349   1.15350   1.16548   1.22059   1.22595
 Alpha virt. eigenvalues --    1.22595   1.56366   1.67175   1.67176   1.80069
 Alpha virt. eigenvalues --    1.84113   1.84114   2.08358   2.12190   2.12191
 Alpha virt. eigenvalues --    2.28546   2.30855   2.30856   2.52068   2.52070
 Alpha virt. eigenvalues --    2.55081   2.59483   2.64270   2.64270   2.67757
 Alpha virt. eigenvalues --    2.67757   3.09697   3.13635   3.85465   3.85467
 Alpha virt. eigenvalues --    4.27526   4.27526   4.36769   4.36769   4.65428
 Alpha virt. eigenvalues --    6.72643   7.68934   7.68934   7.79156   9.86852
 Alpha virt. eigenvalues --   24.03759  26.07802  26.07804  26.18018  48.02020
 Alpha virt. eigenvalues --  215.83126 289.88890 289.88890 290.009601020.79720
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.054469   0.409212   0.409213   0.409232   0.102031  -0.043743
     2  H    0.409212   0.526891  -0.035871  -0.035874  -0.053348   0.009729
     3  H    0.409213  -0.035871   0.526888  -0.035875  -0.053347   0.009729
     4  H    0.409232  -0.035874  -0.035875   0.526846  -0.053340   0.009728
     5  Br   0.102031  -0.053348  -0.053347  -0.053340  35.345305   0.007492
     6  Cl  -0.043743   0.009729   0.009729   0.009728   0.007492  17.909660
 Mulliken charges:
               1
     1  C   -0.340415
     2  H    0.179261
     3  H    0.179262
     4  H    0.179282
     5  Br  -0.294794
     6  Cl  -0.902595
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.197389
     5  Br  -0.294794
     6  Cl  -0.902595
 Electronic spatial extent (au):  <R**2>=           1232.1994
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             11.2262    Y=             -0.1843    Z=             -1.4430  Tot=             11.3201
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -98.9839   YY=            -43.8287   ZZ=            -44.7254
   XY=              0.9045   XZ=              7.0913   YZ=             -0.1163
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -36.4713   YY=             18.6840   ZZ=             17.7872
   XY=              0.9045   XZ=              7.0913   YZ=             -0.1163
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            241.8664  YYY=             -0.3023  ZZZ=             -9.6266  XYY=             23.8788
  XXY=             -3.1968  XXZ=            -24.9732  XZZ=             26.5987  YZZ=             -1.2915
  YYZ=             -2.9767  XYZ=              0.2506
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -2127.0257 YYYY=            -70.5422 ZZZZ=            -95.4685 XXXY=             21.2676
 XXXZ=            166.6151 YYYX=             11.9907 YYYZ=             -1.5388 ZZZX=             98.9505
 ZZZY=             -1.8024 XXYY=           -277.1835 XXZZ=           -289.9537 YYZZ=            -27.6500
 XXYZ=             -1.5447 YYXZ=             32.5602 ZZXY=              4.7535
 N-N= 1.739344965657D+02 E-N=-7.699041276479D+03  KE= 3.069982095725D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE386\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.39811,0.000005,-0.000178\H,0,-0.709489,-0.879048,-0.5472\H,0,-0
 .709487,0.913228,-0.488023\H,0,-0.709674,-0.034154,1.034561\Br,0,1.635
 528,-0.000012,0.000056\Cl,0,-3.493125,0.000025,-0.000045\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.3948736\RMSD=1.248e-09\Dipole=4.453657
 7,-0.0000274,-0.0000546\Quadrupole=-27.8041785,13.9020489,13.9021296,0
 .0002731,-0.0003926,0.0000059\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 A MAN SHOULD NEVER BE ASHAMED TO OWN HE HAS BEEN IN THE WRONG
 WHICH IS BUT SAYING IN OTHER WORDS,
 THAT HE IS WISER TODAY THAN HE WAS YESTERDAY.

                          -- JONATHAN SWIFT
 Job cpu time:       0 days  0 hours  0 minutes 29.9 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.7 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:10 2020.
